Source/Core/Common/StringUtil.cpp

```diff
@@ -230,6 +230,16 @@ std::string StringFromInt(int value)
 	return temp;
 }
 
+std::string StringFromFloat(float value)
+{
+	return StringFromFormat("%.9g", value);
+}
+
+std::string StringFromDouble(double value)
+{
+	return StringFromFormat("%.17g", value);
+}
```

Comment:
That's still kind of pointless. Also the function name doesn't actually indicate that additional formatting is done, which IMO, should be handled by direct calls with StringFromFormat.